dashboard/test/controllers/api/v1/pd/workshop_organizers_controller_test.rb

```diff
@@ -26,5 +27,10 @@ class Api::V1::Pd::WorkshopOrganizersControllerTest < ::ActionController::TestCa
       }.stringify_keys
       assert_equal expected, response.find {|o| o['id'] == organizer.id}
     end
+
+    # No non-organizers returned
+    non_organizers.each do |non_organizer|
+      refute response.any? {|o| o['id'] == non_organizer.id}
+    end
```

Comment:
This test is seemingly unrelated, but started failing consistently with these other changes. It was assuming too much external context anyway, and this is a better way of doing it. Instead of checking that the only organizers are exactly the the 2 we created in the test, instead verify that both of those and none of the known non-organizers are returned but be tolerant of additional (unrelated, external) organizers in the set.